flux_led/__main__.py

```diff
@@ -608,14 +646,25 @@ def update_state(self, retry=2):
 
         # Devices that actually support rgbw
         if (rx[1] == 0x04 or
+            rx[1] == 0x25 or
             rx[1] == 0x81):
             self.rgbwcapable = True
 
         # Devices that use an 8-byte protocol
-        if (rx[1] == 0x27 or
+        if (rx[1] == 0x25 or
+            rx[1] == 0x27 or
             rx[1] == 0x35):
             self.protocol = "LEDENET"
 
+        # Devices that support RGBW, but only as two separate commands
+        if rx[1] == 0x25:
+            self.badrgbw = True
+
+        # Devices that use the 11-byte protocol
```

Comment:
This is a bit confusing - elsewhere we refer to an 8-byte protocol, but that's the length of the commands, not the length of the status messages. From that perspective, this is a 5-byte protocol.